dev/tasks/download-e2e-logs

```diff
@@ -46,3 +46,6 @@ gsutil cp gs://cnrm-prow/logs/${TEST}/${LATEST}/artifacts/golden.zip .build/gold
 
 echo "Expanding golden output into testdata directory"
 unzip -o .build/golden.zip "pkg/test/resourcefixture/testdata/*"
+
+echo "Downloading test log to e2e.log"
+gsutil cp gs://cnrm-prow/logs/${TEST}/${LATEST}/build-log.txt e2e.log
```

Comment:
suggest adding a timestamp to the downloaded file name.

Reply:
The nice thing about going to the same file is that we can grep it easily from other scripts, and that we don't end up with dozens of e2e log files because they overwrite each other.

I think also because the file still exists on GCS, we're not going to lose data we care about by overwriting.

Reply:
fair enough.